jbi/retry.py

```diff
@@ -67,8 +75,12 @@ async def retry_failed(item_executor=runner.execute_action, queue=get_dl_queue()
                         metrics["events_skipped"] += pending_events - 1
                         break
         except Exception:
-            logger.exception("failed to parse events for bug %d.", bug_id)
             metrics["bugs_failed"] += 1
+            logger.exception(
+                "failed to parse events for bug %d.",
+                bug_id,
+                extra={"metrics": metrics, "bug": {"id": bug_id}},
```

Comment:
Same here, with the questions about metrics being included in individual log lines